src/libraries/Microsoft.PowerFx.Core/Public/IntellisenseOperations.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.Parser;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Intellisense
+{
+    /// <summary>
+    /// Various IntelliSense-like operations.
+    /// </summary>
+    public class IntellisenseOperations
+    {
+        private readonly CheckResult _checkResult;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="IntellisenseOperations"/> class.
+        /// </summary>
+        /// <param name="result"></param>
+        public IntellisenseOperations(CheckResult result)

```

Comment:
Have no idea whether this is what we want?

Reply:
This is ok.